tests/baselines/reference/getEmitOutput-pp.baseline

```diff
@@ -1,34 +1,34 @@
 EmitSkipped: false
 
 FileName : /tests/cases/fourslash/shims-pp/inputFile1.js
-var x = 5;
-var Bar = /** @class */ (function () {
-    function Bar() {
-    }
-    return Bar;
-}());
+var x = 5;
+var Bar = /** @class */ (function () {
+    function Bar() {
+    }
+    return Bar;
+}());
```

Comment:
What is happening here? Are these files mixed newlines now?

Reply:
Ugh, yes.
```
> fs.readFileSync("tests/baselines/reference/getEmitOutput-pp.baseline")
<Buffer 45 6d 69 74 53 6b 69 70 70 65 64 3a 20 66 61 6c 73 65 0d 0a 0d 0a 46 69 6c 6
```
Looking into it...

Reply:
Seems like we agreed that this file being mixed is the "right" behavior

Reply:
Guess we'll see what happens the next time someone pulls. Probably fine because our gitattributes says:

```
*.js linguist-language=TypeScript
* -text
```

Aka "nothing is text"